### PR TITLE
Added OTAWebupdated

### PR DESCRIPTION
**Pull request by karan6190**
Added example related to  OTA through Web Page
User can able to update the firmware by uploading the binary file of the code through Web page (user and device must be same LAN )
esp32login
esp32upload

### CODE REVIEW

File: libraries/ArduinoOTA/examples/OTAWebUpdater/OTAWebUpdater.ino [hunk 1]
```diff
@@ -0,0 +1,160 @@
+#include <WiFi.h>
+#include <WiFiClient.h>
+#include <ESP32WebServer.h>
```
Comment (me-no-dev) [inline libraries/ArduinoOTA/examples/OTAWebUpdater/OTAWebUpdater.ino]: it is just WebServer. No ESP32 in front

File: libraries/ArduinoOTA/examples/OTAWebUpdater/OTAWebUpdater.ino [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+#include <ESPmDNS.h>
+#include <Update.h>
+
+const char* host = "ESP32";
+const char* ssid = "xxx";
+const char* password = "xxxx";
+
+ESP32WebServer server(80);
```
Comment (me-no-dev) [inline libraries/ArduinoOTA/examples/OTAWebUpdater/OTAWebUpdater.ino]: same here

File: libraries/ArduinoOTA/examples/OTAWebUpdater/OTAWebUpdater.ino (hunk 1 above)
Comment (karan6190) [reply libraries/ArduinoOTA/examples/OTAWebUpdater/OTAWebUpdater.ino]: yes, WebServer is required for Zero config routing protocol (http://hostname.local), just to detect your device in LAN connectivity. if WebServer not used then everytime the user has to detect Ip of esp32 
And eventually, Firmware will be uploaded in esp32 only.

Comment (me-no-dev) [reply libraries/ArduinoOTA/examples/OTAWebUpdater/OTAWebUpdater.ino]: Yes... but the name of the lib is not ESP32WebServer, but just WebServer ;) I get why it's needed :D

Comment (me-no-dev) [reply libraries/ArduinoOTA/examples/OTAWebUpdater/OTAWebUpdater.ino]: https://github.com/espressif/arduino-esp32/blob/master/libraries/WebServer/src/WebServer.h

**Comment by me-no-dev** [comment]
Thanks :)

**Comment by JoanMCD** [comment]
The user and PW is visible in the html page... (rightclick, 'view page source' in Browser). It suggests a false security !

**Pull request merged**